src/libs/actions/Session/index.js

```diff
@@ -284,13 +284,12 @@ function signInWithShortLivedToken(accountID, email, shortLivedToken) {
             accountID,
             email,
         });
-        if (response.jsonCode === 200) {
-            User.getUserDetails();
-            Onyx.merge(ONYXKEYS.ACCOUNT, {success: true});
-        } else {
-            const error = lodashGet(response, 'message', 'Unable to login.');
-            Onyx.merge(ONYXKEYS.ACCOUNT, {error});
```

Comment:
@aldo-expensify I'm just returning early now instead of setting the error again because it's already set here:

https://github.com/Expensify/App/blob/411e59cd0bf5d2498153a165adc2d81b44f0dba7/src/libs/actions/Session/index.js#L199-L202

I think it's fine? The error message here is not translated anyway and doesn't provide much information so I think better to use the one from the server for now.

Reply:
I think it is fine. The only remaining thing is to remove an unused import (lodash).

Reply:
Thanks!